charts/cmc-claim-store/values.preview.template.yaml

```diff
@@ -46,8 +46,11 @@ java:
     CMC_DB_USERNAME: cmc
     CMC_DB_PASSWORD: cmc
     CORE_CASE_DATA_API_URL: http://${SERVICE_NAME}-data-store-api
-    FEATURE_TOGGLES_CCD_ENABLED: false
-    FEATURE_TOGGLES_CCD_ASYNC_ENABLED: true
+    FEATURE_TOGGLES_CCD_ENABLED: true
+    FEATURE_TOGGLES_CCD_ASYNC_ENABLED: false
```

Comment:
As discussed, we are OK with that until we find a fix that allows us to turn that back on while also having CCD on. At least with this PR we can start exercising most tests agains CCD. Nearly there - good job 👍

Reply:
👍